livestyled/schemas/product.py

```diff
@@ -12,15 +12,6 @@
 from livestyled.schemas.fulfilment_point import FulfilmentPointSchema
 
 
-class ProductVariantStocksSchema(Schema):
```

Comment:
why was this removed? We still have old lambdas making use of this? what if they update the Python version?

Reply:
I don't think that old lambdas could use this, because it wasn't working, it just throws and error or returns empty result. Implementation of this was wrong, since it was waiting for product_variant_stock id, but this table doesn't have id at all, we always have just iri with such structure in the end :  `fulfilmentPoint=1;productVariant=5`

So that's why I changed it, it wasn't working at all